tests/src/test/scala/cats/collections/HeapSpec.scala

```diff
@@ -1,16 +1,19 @@
 package cats.collections
 package tests
 
+import cats.{Order, Show}
 import cats.tests.CatsSuite
+import org.scalacheck.{Arbitrary, Gen}
 
 /**
  * Created by nperez on 3/28/16.
  */
 class HeapSpec extends CatsSuite {
-  test("sorted")(
-    forAll { (xs: scala.List[Int]) =>
+  implicit val propConfig =
+    PropertyCheckConfig(minSuccessful = 200)
```

Comment:
I recommend dialing that up. Some errors in `Diet` (#150) were only found after thousands of iterations.

```suggestion
    PropertyCheckConfig(minSuccessful = if (Platform.isJvm) PosInt(10000) else PosInt(500))
```

Reply:
It takes about 10s I think at 200, at 10000 this would be about 5 minutes of testing.

Are you sure you want that long?

I think the problem is the Gen is pretty expensive to try to simulate Heaps at all possible states. I think that needs to be improved if we should turn it up.

How much of a time budget would you allow for this test?

Reply:
5 minutes would indeed be excessive. Let's say 500 on the JVM and 100 on JS, then.